python/lsst/display/firefly/firefly.py

```diff
@@ -146,11 +148,18 @@ def _mtv(self, image, mask=None, wcs=None, title=""):
                 fd.seek(0,0)
                 self._fireflyFitsID = _fireflyClient.upload_data(fd, 'FITS')
 
+            extraParams = dict(Title=title,
+                               MultiImageIdx=0,
+                               PredefinedOvelayIds=' ')
+                        # Firefly's Javascript API requires a space for parameters;
+                        # otherwise the parameter will be ignored
+
+            if self._lastZoom:
+                extraParams['InitZoomLevel'] = self._lastZoom
+                extraParams['ZoomType'] = 'LEVEL'
```

Comment:
Should there be a similar block for panning here, too?

Reply:
We don't have the right API right now so it will be added once we have that on the Firefly end.